### PR TITLE
WIP: Replace AkkaParadoxPlugin with PekkoParadoxPlugin

### LINKED ISSUE #11: change docs to use pekko paradox plugin
* see https://github.com/apache/incubator-pekko-connectors-samples/blob/main/docs/build.sbt
* have a look at https://github.com/apache/incubator-pekko to see how it sets up the PekkoParadoxPlugin

### PR DESCRIPTION
**Pull request by reypader**
Fixes #11

### CODE REVIEW

**Comment by reypader** [comment]
Base template applies but subpages lose the pekko logo and the header is blue. Still trying to figure out why

File: docs/project/plugins.sbt [hunk 1]
```diff
@@ -1,6 +1,7 @@
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.44")
+addSbtPlugin("org.apache.pekko" % "pekko-sbt-paradox" % "0.0.0+55-2e9e22a8-SNAPSHOT") // #TODO: user release version
```
Comment (pjfanning) [inline docs/project/plugins.sbt:1]: you need to copy paste all of this

https://github.com/apache/incubator-pekko/blob/main/project/plugins.sbt#L39-L47

**Comment by pjfanning** [comment]
@reypader applying this is actually hard. I didn't raise this and expect you to do it. Someone who has done it before will find this much easier. As well as the plugin.sbt issue I already raised, there are quite a lot of settings in incubator-pekko project/Paradox.scala that need to be copied over - the pekkoThemeSettings and those settings need to be wired into the project settings.

I can have a look at this myself in a few days.

**Comment by reypader** [comment]
@pjfanning thanks for the heads up! I'll close this ticket now. Looking at it upfront, it seemed easy enough 😆 . 

>  ...there are quite a lot of settings in incubator-pekko project/Paradox.scala that need to be copied over - the pekkoThemeSettings and those settings need to be wired into the project settings

I was actually staring at this class as well and half considered copying it over!

**Pull request closed without merging**